src/nvim/tui/tui.c

```diff
@@ -354,10 +358,13 @@ static void tui_resize(UI *ui, int width, int height)
   data->scroll_region.right = width - 1;
   data->row = data->col = 0;
 
-  // try to resize the terminal window
-  char r[16];  // enough for 9999x9999
-  snprintf(r, sizeof(r), "\x1b[8;%d;%dt", height, width);
-  out(ui, r, strlen(r));
+  // Try to resize the terminal window
+  // (only if not being resized already on SIGWINCH)
+  if (!during_winch) {
+    char r[16];  // enough for 9999x9999
+    snprintf(r, sizeof(r), "\x1b[8;%d;%dt", height, width);
+    out(ui, r, strlen(r));
```

Comment:
Can it? Since the resize is deferred in `sigwinch_cb`, I wasn't sure `during_winch` would be reset properly.